app/cmd/config_generate.go

```diff
@@ -84,7 +84,8 @@ func printCfg(data []byte) {
 
 func generateSampleConfig() ([]byte, error) {
 	sampleConfig := Config{
-		Current: "yourServer",
+		Current:  "yourServer",
+		Language: "",
```

Comment:
`Language: "",` Can we just remove it?

Reply:
Of course. But I think if don't show it, nobody knows this function

Reply:
It still cannot resolve the problem that you mentioned.

Reply:
Yes. Maybe we can show the example for this. Like this:
`#Language context is accept-language for HTTP header, It contains zh-CN/zh-TW/en/en-US/ja and so on`

Reply:
You can remove `Language: "",`. And please make sure you've tested it by manual. The following was my test result  after I remove it:
```
localhost:jenkins-cli mac$ ./bin/darwin/jcli config generate
current: yourServer
language: ""
jenkins_servers:
- name: yourServer
  url: http://localhost:8080/jenkins
  username: admin
  token: 111e3a2f0231198855dceaff96f20540a9
  proxy: ""
  proxyAuth: ""
  description: ""
preHooks: []
postHooks: []
pluginSuites: []
# Language context is accept-language for HTTP header, It contains zh-CN/zh-TW/en/en-US/ja and so on
# Goto 'http://localhost:8080/jenkins/me/configure', then you can generate your token.
```

So giving it an empty value is not necessary.